package.json

```diff
@@ -28,7 +28,7 @@
   "scripts": {
     "build": "babel *.js -d lib",
     "lint": "eslint --fix *.js test",
-    "pretest": "eslint *.js test",
+    "posttest": "eslint *.js test",
```

Comment:
Recently someone turned me on to the fact you should just be able to do:

```
"lint": "eslint ."
```

Reply:
This would cause it to also lint against the `lib/` directory. Should we go with the exclusion route rather than the inclusion route?